keywording/text.xml

```diff
@@ -267,6 +291,38 @@ Vulnerability Treatment Policy</uri>
 
 </body>
 
+<subsection>
+<title>Stabilization rules</title>
+<body>
+
+<p>
+SPARC: You must have prior permission from the arch lead. Usually we expect
+you to be on the sparc alias for QA reasons, although other arrangements
+can be made if you will only be working with a small group of packages.
+</p>
+
+<p>
+ALPHA: Maintainers may keyword their own packages but are reminded to inform
+the Alpha team if they can help out with testing and keywording packages so
+the team can keep an eye out for possible keywording mistakes.
+</p>
+
+<p>
+Exotic architectures (like alpha, ia64, sparc, hppa, ppc*) are short
+on manpower, so it's best if you avoid opening stabilization bugs for
+them unless it is absolutely necessary (eg, a reverse dependency for
+your package).
+</p>
+
+<p>
+Some architectures (like m68k, mips, s390, sh) do not maintain a
+stable keyword. So there is no use in marking a package stable for one
```

Comment:
"do not maintain stable profiles" maybe?

Reply:
I think it's keyword here, because stability of the profile doesn't necessarily coincide with it.

Reply:
...but `m68k` has stable keywords (:wave: vapier)?

Reply:
Changed to say only mips, riscv.